cloud/google/gce.py

```diff
@@ -87,10 +87,18 @@
       - your GCE project ID
     required: false
     default: null
-  name:
+  base_name:
```

Comment:
@ryansb: @supertom said he wanted to move away from ```name``` so that was intentional. I am happy to add an alias and have ```base_name``` be the primary invocation to move away from name per Tom's request.